api/lib/validation/schemaValidator.js

```diff
@@ -93,7 +93,7 @@ var getScalingRuleSchema = function() {
       'metric_type':{ 'type':'string' ,'enum':metricTypeEnum },
       'stat_window_secs':{ 'type':'number','minimum': 60,'maximum': 3600 },
       'breach_duration_secs':{ 'type':'number','minimum': 60,'maximum': 3600 },
-      'threshold':{ 'type':'number','minimum': 1,'maximum': 100 },
```

Comment:
I know which is why I am asking why we removed the minimum. If it is > 0, we must have a value which is >= 1. Rather than code the check above, why not have the validator deal with it?